.github/workflows/ci.yml

```
@@ -49,6 +49,10 @@ jobs:
           libraries: ''
 
   online:
+    # Don't run online tests for tagged releases
+    if: |
+      github.event_name == 'pull_request' &&
+          !startsWith(github.event.ref, 'refs/tags/v')
```

Including the PR check would mean online tests would only run during PRs, and not during merges or crons.
```suggestion
    if: !startsWith(github.event.ref, 'refs/tags/v')
```